index.html

```diff
@@ -184,10 +184,10 @@ <h2 class="headline">
               <div class="social-handles">
                   <p>Let's Connect</p>
                   <ul class="social-icons">
-                      <li class="github"><a href="https://github.com/ishpaul777"><img src="./assets/icons/github.svg" alt="github"></a></li>
-                      <li class="linkedin"><a href="https://www.linkedin.com/in/ishpaul-singh-264590226/"><img src="./assets/icons/linkedin.svg" alt="linkedin"></a></li>
-                      <li class="twitter"><a href="https://twitter.com/ishpaul_777"><img src="./assets/icons/twitter.svg" alt="twitter"></a></li>
-                      <li class="instagram"><a href="https://www.instagram.com/ishpaul_777"><img src="./assets/icons/instagram.svg" alt="instagram"></a></li>
+                      <li class="github"><a href="https://github.com/ishpaul777"><img src="./assets/icons/github.svg" alt="github-icon"></a></li>
+                      <li class="linkedin"><a href="https://www.linkedin.com/in/ishpaul-singh-264590226/"><img src="./assets/icons/linkedin.svg" alt="linkedin-icon"></a></li>
+                      <li class="twitter"><a href="https://twitter.com/ishpaul_777"><img src="./assets/icons/twitter.svg" alt="twitter-icon"></a></li>
+                      <li class="instagram"><a href="https://www.instagram.com/ishpaul_777"><img src="./assets/icons/instagram.svg" alt="instagram-icon"></a></li>
                   </ul>
               </div>
               <button type="button" class="resume-btn">Get my resume</button>
```

Comment:
- [ ] please the `languages` is a subheading and should be in an `h `tag not a `p tag.` kindly fix that